presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -0,0 +1,228 @@
+=================
+Dynamic Filtering
```

Comment:
should we update to 

Dynamic filtering and dynamic partition pruning

??

Reply:
I think it's ok for the title to be "Dynamic filtering" as partition pruning is just a special case of DF for Hive. We'll also eventually have other types of dynamic filtering like dynamic bucket pruning for bucketed Hive tables and dynamic row filtering (compiled filters after table scan). So the umbrella term of dynamic filtering should be ok as the title.

Reply:
Sounds good. we can have separate subsections for things later .. for now this is good